editor.planx.uk/src/ui/editor/TemplatedNodeInstructions.tsx

```diff
@@ -0,0 +1,28 @@
+import StarIcon from "@mui/icons-material/Star";
+import Typography from "@mui/material/Typography";
+import { BaseNodeData } from "@planx/components/shared";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+
+import ModalSection from "./ModalSection";
+import ModalSectionContent from "./ModalSectionContent";
+
+export const TemplatedNodeInstructions = ({
+  isTemplatedNode,
+  templatedNodeInstructions,
+  areTemplatedNodeInstructionsRequired,
+}: BaseNodeData) => {
+  const isTemplatedFrom = useStore.getState().isTemplatedFrom;
+  if (!isTemplatedFrom || !isTemplatedNode) return null;
+
+  return (
+    <ModalSection sectionBackgroundColor="#e6d6ff">
+      <ModalSectionContent
+        title={`Instructions ${areTemplatedNodeInstructionsRequired ? `(required)` : `(optional)`}`}
+        Icon={StarIcon}
+      >
+        <Typography variant="body2">{templatedNodeInstructions}</Typography>
+      </ModalSectionContent>
+    </ModalSection>
```

Comment:
I thought about using an existing `WarningContainer` here, but ultimately preferred ability to keep title and icon aligned using `ModalSectionContent` - think it's a bit tidier and more prominent.

Title of this section is another interesting one - I also thought of: 
- "Templated node instructions" - but essentially I think we want purple to communicate "templated" for us, so we can just say "Instructions"
- "Instructions*" or "Instructions" - but decided to spell out "(required)" and "(optional)" instead as we don't use the `*` elsewhere in input labels etc

Reply:
Updated - good shout!